src/styles/utils/index.ts

```diff
@@ -1522,6 +1522,37 @@ const createStyleUtils = (theme: ThemeColors, styles: ThemeStyles) => ({
         return isDragging ? styles.cursorGrabbing : styles.cursorZoomOut;
     },
 
+    getSearchTableColumnStyles: (columnName: string): ViewStyle => {
+        let columnWidth;
+        switch (columnName) {
+            case CONST.SEARCH_TABLE_COLUMNS.DATE:
+                columnWidth = {width: 44};
```

Comment:
all of these CONST widths should be added to `variables.ts`

Reply:
Done